schema-manager/src/test/java/io/camunda/search/schema/SchemaManagerIT.java

```diff
@@ -786,4 +789,55 @@ void shouldCreateHarmonizedSchema(
             newPrefix + "-tasklist-import-position-8.2.0_",
             newPrefix + "-tasklist-user-1.4.0_");
   }
+
+  @TestTemplate
+  void shouldRecordSchemaInitTimerMetric(
+      final SearchEngineConfiguration config, final SearchClientAdapter ignored) {
+    // given
+    final var schemaManagerMetrics = mock(SchemaManagerMetrics.class);
+    final var timer = mock(CloseableSilently.class);
+    when(schemaManagerMetrics.startSchemaInitTimer()).thenReturn(timer);
```

Comment:
🔧 Instead of mocking and rely on the internal methods and calls, could we also validate whether metrics are exposed? So actually testing behavior instead of internals.